static/js/util/profile_edit.js

```diff
@@ -62,7 +69,7 @@ export function boundRadioGroupField(keySet: string[], label: string, options: O
     );
   });
 
-  const value = _.get(profile, keySet);
+  const value = String(_.get(profile, keySet));
```

Comment:
it is giving me 3 warning messages if I use boolean. here

- https://github.com/mitodl/micromasters/blob/a235e8d3292c708f88d82637e9bc38f14fa5dc9f/static/js/util/profile_edit.js#L64
- https://github.com/mitodl/micromasters/blob/a235e8d3292c708f88d82637e9bc38f14fa5dc9f/static/js/util/profile_edit.js#L66
- https://github.com/mitodl/micromasters/blob/a235e8d3292c708f88d82637e9bc38f14fa5dc9f/static/js/util/profile_edit.js#L82